### ISSUE TITLE
Theme howto snippet xpath is wrong

### ISSUE THREAD

**Issue by lslaz1** [odoo/odoo#12523]
In the "Snippet" section of doc/howtos/themes.rst is says to add 
`<xpath expr="//div[@id='snippet_structure']" position="inside">`
However, this will only add the content of the snippet into the Website Builder. 
The code should read
```
<xpath expr="//div[@id='snippet_structure']/div[contains(@class, 'o_panel_body')]" position="inside">
```
so that the drag&drop thumbnail is inserted instead of the content.

**Comment by Yenthe666** [comment]
@JKE-be another one that might be for you. Can you have a look at this?

**Comment by mart-e** [comment]
Thanks, fixed at cecc12ffc6a26ae61319b358f69d9fc5e0ad5bcc

@lslaz1 as you already had the correction, don't hesitate to create a pull request for next time

**Comment by lslaz1** [comment]
Thanks, I will definitely do that next time
